event_product_logistics/models/product.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    event_deliverable_product_ids = fields.Many2many(
```

Comment:
Why don't you add this to `event.event` instead.
With that you could have this module depend directly on `event` instead of `event_product`